GVRf/Framework/backend_oculus/build.gradle

```diff
@@ -136,8 +132,6 @@ android {
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
     compile fileTree(include: ['*.jar'], dir: 'src/main/libs')
-    compile(name: 'google-vr-base', ext: 'aar')
-    compile(name: 'google-vr-common', ext: 'aar')
     compile project(':framework')
 }
 

```

Comment:
Cant comment on the next file, will add it here instead:

Why is the oculus backend being loaded at order 2?